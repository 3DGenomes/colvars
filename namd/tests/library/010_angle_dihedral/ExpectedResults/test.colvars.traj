# step         angle                 fs_angle              fa_angle               dihedral              fs_dihedral           fa_dihedral           
           0    9.68498695196423e+01  0.00000000000000e+00  0.00000000000000e+00   4.86030392835581e+01  0.00000000000000e+00  0.00000000000000e+00 
           1    9.68581311788660e+01 -1.84221082305878e-02  1.84221082305878e-02   4.85122724853581e+01  2.72342642214669e-02 -2.72342642214669e-02 
           2    9.68826318576805e+01 -3.61703145887794e-02  2.72962114096836e-02   4.84188109435140e+01  6.33375772744182e-02 -4.52859207479426e-02 
           3    9.69107048283130e+01 -6.12849283114021e-02  3.86257837102564e-02   4.83277750302569e+01  8.88671420729904e-02 -5.98129945229585e-02 
           4    9.69409611522252e+01 -1.28333888758329e-01  6.10528099722745e-02   4.82463676443870e+01  7.85439066964011e-02 -6.44957225663192e-02 
           5    9.69681353268796e+01 -2.06771099527608e-01  9.01964678833413e-02   4.81713072850571e+01  4.82520664595261e-02 -6.12469913449606e-02 
           6    9.69963407789603e+01 -2.82889570786954e-01  1.22311985033943e-01   4.80948431446689e+01  3.37716513114058e-03 -5.16020203093239e-02 
           7    9.70286631775983e+01 -3.42230881518776e-01  1.53728970246062e-01   4.80201747278237e+01 -3.83338585648162e-02 -3.87540376130182e-02 
           8    9.70554213175582e+01 -3.90720549632060e-01  0.00000000000000e+00   4.79553156452261e+01 -8.22138552821131e-02  0.00000000000000e+00 
           9    9.70978055473150e+01 -3.97503412484494e-01  3.97503412484494e-01   4.78902716335418e+01 -1.21521954248646e-01  1.21521954248646e-01 
          10    9.71536473973138e+01 -4.30993372178144e-01  4.14248392331319e-01   4.78184376211816e+01 -1.83318320789132e-01  1.52420137518889e-01 
