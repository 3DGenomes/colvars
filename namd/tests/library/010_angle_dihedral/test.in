colvarstrajfrequency    1
colvarsrestartfrequency 2000

colvar {
  name angle

  outputSystemForce on
  outputAppliedForce on
  width 6.0 

  lowerBoundary    0.0
  upperBoundary  180.0

  angle {
    group1 {
      psfSegID BH
      atomNameResidueRange CA 1-1
    }
    group2 {
      psfSegID BH
      atomNameResidueRange CA 2-2
    }
    group3 {
      psfSegID BH
      atomNameResidueRange CA 3-3
    }
  }
} 


colvar {
  name dihedral

  outputSystemForce on
  outputAppliedForce on
  width 6.0 

  lowerBoundary    0.0
  upperBoundary  180.0

  dihedral {
    group1 {
      psfSegID BH
      atomNameResidueRange CA 1-1
    }
    group2 {
      psfSegID BH
      atomNameResidueRange CA 2-2
    }
    group3 {
      psfSegID BH
      atomNameResidueRange CA 3-3
    }
    group4 {
      psfSegID BH
      atomNameResidueRange CA 4-4
    }
  }
} 


abf {
    colvars angle dihedral
    fullSamples 1
}

