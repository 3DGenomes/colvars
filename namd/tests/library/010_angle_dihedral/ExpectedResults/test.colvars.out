colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013 in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = angle
colvars:   Initializing a new "angle" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "group1".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group3".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group3" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:   All components initialized.
colvars:   # width = 6
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 180
colvars:   # upperWallConstant = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = dihedral
colvars:   Initializing a new "dihedral" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:       Initializing atom group "group1".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group3".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group3" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group4".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group4" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:   All components initialized.
colvars:   # width = 6
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 180
colvars:   # upperWallConstant = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { angle, dihedral }
colvars:   # outputEnergy = off [default]
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 1
colvars:   # inputPrefix =  [default]
colvars:   # outputFreq = 2000 [default]
colvars:   # historyFreq = 10
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (4 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group angle:0/0. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group angle:0/1. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group angle:0/2. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group dihedral:0/0. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group dihedral:0/1. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group dihedral:0/2. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group dihedral:0/3. 1 atoms: total mass = 12.011.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (4 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars:   ABFHISTORYFILE 0
colvars:   Prepared sample and gradient buffers at step 0.
colvars:   ABFHISTORYFILE 10
colvars: Saving collective variables state to "test.colvars.state".
